src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  notification: string;
+  onSetNotification: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const Notification: React.FC<Props> = React.memo(({
+  notification,
+  onSetNotification,
+}) => {
+  const wait = setTimeout(() => onSetNotification(''), 3000);
```

Comment:
`wait` it's not a good name. Try smth more clearable. notificationDelay for example